tests/ignite/test_engine.py

```diff
@@ -68,7 +61,7 @@ def test_adding_multiple_event_handlers():
 
     engine.run(1)
     for handler in handlers:
-        handler.assert_called_once_with(engine)
+        handler.assert_called_once()
```

Comment:
This should be called with the state right?

Reply:
good catch